### PR TITLE
add logout page -- new, clean PR

### PR DESCRIPTION
**Pull request by cjdufault**
@yp0975eu @NateBigStone @ashley7295 @sunnycake 
Closes #24

### CODE REVIEW

File: lmn/templates/lmn/base.html [hunk 1]
```diff
@@ -14,11 +15,19 @@ <h1>LMN</h1>
 <a href="{% url 'latest_notes' %}">Notes</a>
 
 {% if user.is_authenticated %}
-  <span id='welcome-user-msg'>You are logged in, <a href="{% url 'my_user_profile' %}">{{ user.username }}</a>.  <a href="{% url 'logout' %}">Logout</a></span>
+  <span id='welcome-user-msg'>You are logged in, <a href="{% url 'user_profile' user_pk=user.pk %}">{{ user.username }}</a>.  <a id="logout"  href="{% url 'logout' %}">Logout</a></span>
```
Comment (yp0975eu) [inline lmn/templates/lmn/base.html:18]: do you have the master branch merged into this branch? 'my_user_profile' is the correct url for this line

**Review by yp0975eu — CHANGES_REQUESTED** [review]
Looks like this may still be behind master branch.

File: lmn/templates/lmn/base.html (hunk 1 above)
Comment (cjdufault) [reply lmn/templates/lmn/base.html:18]: Oh, you know, I copied that from the other branch, and didn't check if it was up do date. I'll make that change.

Comment (cjdufault) [reply lmn/templates/lmn/base.html]: Done.

**Review by yp0975eu — APPROVED** [review]

**Review by NateBigStone — APPROVED** [review]

**Comment by sunnycake** [comment]
An operational error when clicking on user profile, but successfully logged out with message!

**Comment by cjdufault** [comment]
@sunnycake I'm not getting that error, can you post the stack trace?

**Comment by NateBigStone** [comment]
@cjdufault @sunnycake You may need to makemigration / migrate to get the new profile table ?

**Pull request merged**